snuba/snuba_migrations/functions/0003_add_new_columns_to_raw_functions.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+from typing import Sequence
+
+from snuba.clickhouse.columns import Column, DateTime64, String
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget
+
+
+@dataclass(frozen=True)
+class NewColumn:
```

Comment:
Is a class really necessary here? I feel like an array of tuples would work just as well.

Reply:
strictly necessary? probably not, but I wanted to be consistent with the other migrations in this same dataset: https://github.com/getsentry/snuba/blob/master/snuba/snuba_migrations/functions/0002_add_new_columns_to_raw_functions.py#L11-L14